src/celeritas/global/Stepper.hh

```diff
@@ -137,27 +138,27 @@ class Stepper final : public StepperInterface
     ~Stepper();
 
     // Transport existing states
-    StepperResult operator()() final;
+    StepperResult operator()() override final;
 
     // Transport existing states and these new primaries
-    StepperResult operator()(SpanConstPrimary primaries) final;
+    StepperResult operator()(SpanConstPrimary primaries) override final;
 
     // Reseed the RNGs at the start of an event for reproducibility
-    void reseed(EventId event_id) final;
+    void reseed(EventId event_id) override final;
 
     //! Get action sequence for timing diagnostics
-    ActionSequence const& actions() const final { return *actions_; }
+    ActionSequence const& actions() const override final { return *actions_; }
 
     //! Access core data, primarily for debugging
     StateRef const& state_ref() const { return state_.ref(); }
 
     //! Get the core state interface for diagnostic output
-    CoreStateInterface const& state() const final { return state_; }
+    CoreStateInterface const& state() const override final { return state_; }
```

Comment:
`override final` is redundant: leave it as `final`. I got this suggestion from 
[the google style guide](https://google.github.io/styleguide/cppguide.html#Inheritance) and it seems nicer to have one keyword instead of two :)

Reply:
The suggestion makes sense. I'd say `override` is less redundant since the class is already declared `final`.

Reply:
Just as redundant, but less consistent. Also less consistent with the code base and the style guides we use. Is this something we should change throughout celeritas? I would need a little more convincing...

Reply:
probably not worth the time, they're pretty much equivalent in a final class.